test/files/pos/nowarnMacros/Test_2.scala

```diff
@@ -0,0 +1,36 @@
+// scalac: -Xfatal-warnings
+
+import language.experimental.macros
+import scala.annotation.nowarn
+
+// FIXME this is meant to be a neg test
```

Comment:
the warnings here don't show up due to https://github.com/scala/scala/pull/8995

the test was modified in 2.13.x to trigger a different warning, see https://github.com/scala/scala/commit/42094282694d3d61d6f0baa7800b59929345485b#diff-85c0d60e1ca722bbccc975ea082b002140f4c5e4be351b8be53ef64b9e1d3844